clients/githubrepo/client.go

```diff
@@ -192,7 +192,14 @@ func (client *Client) ListSuccessfulWorkflowRuns(filename string) ([]clients.Wor
 
 // ListCheckRunsForRef implements RepoClient.ListCheckRunsForRef.
 func (client *Client) ListCheckRunsForRef(ref string) ([]clients.CheckRun, error) {
-	return client.checkruns.listCheckRunsForRef(ref)
+	if crs, ok := client.graphClient.checkRuns[ref]; ok {
```

Comment:
Nit: IMO, cross-file var references reduce code readability - you need context from the other file to understand what the var is. Consider replacing with `graphClient.listCheckRunsForRef()` or similar self-explanatory fn name instead.

Reply:
Is the update any better?

Reply:
LGTM.